### ISSUE TITLE
after upgrade to Hassio 0.96.5 CCH does not work in Chrome anymore

### ISSUE THREAD

**Issue by papperone** [maykar/compact-custom-header#179]
## Describe the issue:
after upgrading to Hass.io 0.96.5 the component stopped working in Chrome

## Config YAML:

```cch:
  clock_date: true
  clock_format: 24
  options: clock
  swipe: true
  swipe_animate: fade
  voice: hide
```
## HA and/or Browser Error:
no error on browser, CCH item in the top/right menu does not appears and as well the header is standard and not anymore "compact"

## Versions and Browser:

CCH = 1.3.1
Hassio = 0.96.5
Browser = Chrome 75.0.3770.100
Lovelace = Storage Mode

![immagine](https://user-images.githubusercontent.com/28104633/62069255-2169b800-b238-11e9-81dc-50a9caf9d9bc.png)

**Comment by maykar** [comment]
I can confirm that CCH is functional in chrome on 0.96.5. If you are not seeing any errors in logs or browser dev tools then it sounds like a caching issue (especially since it's working in FF). Did you install manually or with HACS? What does your resources section look like?

**Comment by papperone** [comment]
yes I did clean up cache and even update Chrome with no luck.
my CCH is manually installed but it was done more than 1 month ago and it was working flawlesssly.
I upgraded CCH from 1.2.x (cannot remember) to 1.3.1 and it was ok then I upgraded hassio from 0.95.4 to 0.96.5 and CCH stopped working on Chrome.
what other test can I do to fix the problem? (it's much better to be able to use Hassio as Chrome App link rather then going back using it from FF) :(
